lib/util/getErrorsFromModelValidation.ts

```diff
@@ -22,18 +22,19 @@ export function getErrorsFromModelValidation(
   }
 
   const entries = sm.entries(validationResult.operations)
-  const operations = it.filterMap(
+  const operationScenarios = it.filterMap(
     entries,
     ([operationId, operation]) => {
-      const examples = operation["x-ms-examples"]
-      if (examples === undefined) {
-        return undefined
-      }
-      const scenarios = examples.scenarios
-      if (scenarios === undefined) {
-        return undefined
-      }
+      const xMsScenarios = operation["x-ms-examples"]
+      const scenario = operation["example-in-spec"]
+      const scenarios = sm.merge(
+        xMsScenarios !== undefined && xMsScenarios.scenarios !== undefined ?
+          xMsScenarios.scenarios :
+          {},
+        scenario !== undefined ? { "example-in-spec": scenario } : {}
```

Comment:
>scenario !== undefined ? { "example-in-spec": scenario } [](start = 8, length = 56)

nit: maybe extract as a vriable to be cleat that we are just adding this as a scenario name: scenario